internal/metrics/metrics.go

```diff
@@ -187,6 +227,172 @@ func GetScalingSignal(pdb *policyv1.PodDisruptionBudget) string {
 	return PDBBlockedSignal
 }
 
+// GetScalingOpportunityLabels gets age pattern and effectiveness labels for scaling opportunities
+func GetScalingOpportunityLabels(ctx context.Context, c client.Client, pdb *policyv1.PodDisruptionBudget) (string, string) {
+	pattern, err := AnalyzePodAgeFailurePattern(ctx, c, pdb)
+	if err != nil {
+		return "unknown", "unknown"
+	}
+	
+	isHelpful := PredictScalingEffectiveness(pattern)
+	helpfulLabel := UnlikelyHelpfulStr
+	if isHelpful {
+		helpfulLabel = LikelyHelpfulStr
+	}
+	
+	return pattern, helpfulLabel
+}
+
+// AnalyzePodAgeFailurePattern analyzes the failure pattern of pods based on their age
+func AnalyzePodAgeFailurePattern(ctx context.Context, c client.Client, pdb *policyv1.PodDisruptionBudget) (string, error) {
+	// Convert PDB label selector to Kubernetes selector
+	selector, err := metav1.LabelSelectorAsSelector(pdb.Spec.Selector)
+	if err != nil {
+		return AllHealthyPattern, err
+	}
+
+	// List all pods matching the PDB selector
+	var podList corev1.PodList
+	err = c.List(ctx, &podList, &client.ListOptions{
+		Namespace:     pdb.Namespace,
+		LabelSelector: selector,
+	})
+	if err != nil {
+		return AllHealthyPattern, err
+	}
+
+	if len(podList.Items) == 0 {
+		return AllHealthyPattern, nil
```

Comment:
this isn't really all healthy this is a different pattern maybe (an abandoned pdb? we mention this elsewhere in the code)